pkg/login/flow.go

```diff
@@ -168,10 +168,13 @@ func (f *Flow) triggerMethod(provider *Provider) (acsToken string, err error) {
 			if err != nil {
 				return "", err
 			}
-			if resp.StatusCode == 401 {
-				err = errors.New("invalid auth token")
-			} else {
+			switch resp.StatusCode {
+			case 200, 403:
```

Comment:
When do we expect a 403 emitted by whom?

What is `acsToken`? Is it the DC/OS authentication token?

The DC/OS authentication token is never emitted within a 403 response.

Reply:
> When do we expect a 403 emitted by whom?
I do not know, this is what we currently have in the Python code. The resource we use to verify the auth token is the same that we use to challenge auth on OSS clusters (`/pkgpanda/active.buildinfo.full.json`).

This is the `browser-prompt-authtoken` so we already have the token even though a 403 response is returned.

Reply:
I've update the comment to use `authentication token` instead of `ACS token`